File: src/cs/AdobeSensei.cs
using System.Net.Http;
using System.Net.Http.Headers;

public class AdobeSenseiAPICall {
    public static async Task Main() {
        string accessToken = "YOUR_ACCESS_TOKEN";
        string apiUrl = "https://platform.adobe.io/data/sensei/mlServices";

        using (HttpClient httpClient = new HttpClient())
        {
            httpClient.DefaultRequestHeaders.Authorization = new AuthenticationHeaderValue("Bearer", accessToken);
            httpClient.DefaultRequestHeaders.Add("x-gw-ims-org-id", "YOUR_ORGANIZATION_ID");
            httpClient.DefaultRequestHeaders.Add("x-api-key", "YOUR_API_KEY");

            HttpResponseMessage response = await httpClient.GetAsync(apiUrl);
            response.EnsureSuccessStatusCode();

            string responseBody = await response.Content.ReadAsStringAsync();
            Console.WriteLine("API Response:");
            Console.WriteLine(responseBody);
        }
    }
}

File: src/java/AdobeSensei.java
import org.apache.http.client.methods.HttpGet;
import org.apache.http.impl.client.CloseableHttpClient;
import org.apache.http.impl.client.HttpClients;
import org.apache.http.util.EntityUtils;

public class AdobeSenseiAPICall {
    public static void main(String[] args) {
        String accessToken = "YOUR_ACCESS_TOKEN";
        String apiUrl = "https://platform.adobe.io/data/sensei/mlServices";

        CloseableHttpClient httpClient = HttpClients.createDefault();
        HttpGet httpGet = new HttpGet(apiUrl);

        httpGet.setHeader("Authorization", "Bearer " + accessToken);
        httpGet.setHeader("x-gw-ims-org-id", "YOUR_ORGANIZATION_ID");
        httpGet.setHeader("x-api-key", "YOUR_API_KEY");

        try {
            HttpResponse response = httpClient.execute(httpGet);
            String responseBody = EntityUtils.toString(response.getEntity());
            System.out.println("API Response:");
            System.out.println(responseBody);
        } catch (IOException e) {
            e.printStackTrace();
        } finally {
            try {
                httpClient.close();
            } catch (IOException e) {
                e.printStackTrace();
            }
        }
    }
}
